src/locales/zh_CN/battle.ts

```diff
@@ -77,6 +77,10 @@ export const battle: SimpleTranslationEntries = {
   "statSeverelyFell": "{{pokemonNameWithAffix}}\n的{{stats}}极大幅降低了！",
   "statWontGoAnyLower": "{{pokemonNameWithAffix}}\n的{{stats}}已经无法再降低了！",
   "ppReduced": "降低了{{targetName}}的\n{{moveName}}的PP{{reduction}}点！",
+  "retryBattle": "你要从对战开始时重试么？",
+  "unlockedSomething": "{{unlockedThing}}\n已解锁。",
+  "congratulations": "恭喜！",
+  "beatModeFirstTime": "{{speciesName}} 首次击败了 {{gameMode}} 模式！\n你获得了{{newModifier}}！",
```

Comment:
```suggestion
  "beatModeFirstTime": "{{speciesName}}首次击败了{{gameMode}}！\n你获得了{{newModifier}}！",
```
we've already included "mode" in the replacement so I deleted “模式”